src/NServiceBus.RabbitMQ/Configuration/DelayedDeliverySettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using Configuration.AdvanceExtensibility;
+    using Settings;
+
+    /// <summary>
+    /// TODO: Fill this out
+    /// </summary>
+    public class DelayedDeliverySettings : ExposeSettings
+    {
+        internal DelayedDeliverySettings(SettingsHolder settings) : base(settings) { }
+
+        /// <summary>
+        /// TODO: Fill this out
+        /// </summary>
+        public void DisableTimeoutManager()
```

Comment:
We thought about it @WojcikMike and came to conclusion that fluent API while feels great is not what we'd need. `DisableTimeoutManager` and `AllEndpointsSupportNativeDelays` are not ordered. Fluent API makes sense when you need an order of operations. That's why we ended up with grouping under the `.DelayedDelivery()` scope. 

@bording please correct me if I'm wrong or missed something.